packages/node/test/unit/install.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  AppABIEncodings,
+  AppState,
+  AssetType,
+  BlockchainAsset
+} from "@counterfactual/types";
+import dotenv from "dotenv-extended";
+import { Wallet } from "ethers";
+import { AddressZero, One, Zero } from "ethers/constants";
+import { instance, mock, when } from "ts-mockito";
+import { v4 as generateUUID } from "uuid";
+
+import { ERRORS, install } from "../../src/methods/install-operations";
+import { openStateChannel } from "../../src/methods/multisig-operations";
+import { ProposedAppInstanceInfo } from "../../src/models";
+import { Store, STORE_ERRORS } from "../../src/store";
+import { EMPTY_NETWORK } from "../integration/utils";
+import MEMORY_STORE_SERVICE from "../services/memory-store-service";
+
+dotenv.load();
+
+describe("Can handle correct & incorrect installs", () => {
+  const storeKeyPrefix = "store";
+
+  it("fails to install without appInstanceId", () => {
+    const store = new Store(MEMORY_STORE_SERVICE, storeKeyPrefix);
```

Comment:
You can instantiate this store in a `beforeAll`

Reply:
Since the second half of the test cases use a mocked store, the `store` variable actually loses uniform typing across the test cases,  so if I were to move anything to a `beforeAll` it'd just be the variable declaration, and there's not much benefit from doing that.

Reply:
I mean, you can always use multiple `describe` blocks

Reply:
I could, though generally I lean towards having one describe block to describe a test suite per file.